app/utility/session.py

```diff
@@ -28,6 +32,11 @@ def __init__(self, services, log):
         agents = await self.services.get('data_svc').locate('agents')
         return next((i for i in agents if i['paw'] == paw), False)
 
+    @staticmethod
+    async def _handshake(reader, writer):
+        recv_proof = await reader.read(32)
+        return recv_proof == b'94699f9970213dd1d4054ca678f1278a'
```

Comment:
let's move to some config file

Reply:
Waiting to do this until https://github.com/mitre/terminal/pull/15 gets in since we'll probably want to bake the config'd value into the shell when we recompile it.